topydo/commands/ListCommand.py

```diff
@@ -101,6 +104,9 @@ def _process_flags(self):
 
                 # when a user requests a specific ID, it should always be shown
                 self.show_all = True
+            elif opt == '-p':
+                self.priorities = [f"(={p})" if len(p) == 1 else p for p
```

Comment:
Based on the tiddlywiki edit, I would expect this to detect a more complex expression.